cfdm/test/test_Domain.py

```diff
@@ -227,12 +227,13 @@ def test_Domain_persist(self):
         """Test Domain.persist."""
         f = cfdm.example_field(0)
         cfdm.write(f, tmpfile)
-        f = cfdm.read(tmpfile)[0].domain
+        f = cfdm.read(tmpfile, dask_chunks=3)[0][:2, :2].domain
 
         on_disk = False
-        for v in f.coordinate("longitude").data.todict().values():
+        for k, v in f.coordinate("longitude").data.todict().items():
             if isinstance(v, cfdm.data.abstract.FileArray):
                 on_disk = True
+                break
```

Comment:
Not needed? I guess this was put in for dev work and not removed?

```suggestion
```

Reply:
It's OK isn't it? No need to iterate through more of the loop than needed?

Reply:
Ah yes of course, sorry, I clearly didn't look carefully enough and though it was an indent level down and skipped everything. My bad. Marking all as resolved.